compiler/src/dotty/tools/dotc/transform/ElimJavaPackages.scala

```diff
@@ -0,0 +1,24 @@
+package dotty.tools.dotc.transform
```

Comment:
It's funny how now with the new github system we can have comments with wrong review dates. See reply in a comment above.